themes/cardinal_service/pdb_components/opportunities_list/src/Components/SelectList.jsx

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react';
+import Select from 'react-select';
+
+const _ = require('lodash-uuid');
+
+export class SelectList extends Component {
+
+  constructor(props) {
+    super(props);
+    this.uuid = _.uuid();
+    this.state = {selectedItems: []};
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange(selectedOptions) {
+    if (selectedOptions === null) {
+      this.props.onChange(this.props.field, []);
+      return;
+    }
+
+    const selections = this.props.multiple ? selectedOptions.map(item => item.value) : [selectedOptions.value];
+    this.props.onChange(this.props.field, selections);
+  }
+
+  filterOptions = (candidate, input) => {
+    if (input) {
+      return candidate.data.label.toLowerCase().indexOf(input.toLowerCase()) >= 0;
+    }
+    return true;
+  };
+
+  render() {
+    if (this.props.options === undefined) {
+      return <React.Fragment/>
```

Comment:
TIL: https://reactjs.org/docs/fragments.html

Reply:
yeah its a very useful thing to keep the markup lower. always gotta have wrapper, but not necessarily a visible one.

Reply:
I've used `<>` in the past but I like this better.